README.md

```diff
@@ -30,3 +30,68 @@ my_metric_with_label_set.labels({'req2': '2'}).inc()  # would work because it ha
 ```
 
 The way to create metrics is up to discussion, instead of a separated function it could be the class itself that creates everything with the appropriate arguments.
+
+## How to use different backends
+
+Things work out of the box, using the SingleProcessBackend:
+
+```python
+
+from pytheus.metrics import create_counter
+
+counter = create_counter(
+    name="my_metric",
+    required_labels=["label_a", "label_b"],
+)
+print(counter._backend.__class__)
+print(counter._backend.config)
```

Comment:
small nitpick: I would add the output of the print calls as comment underneath maybe 🤔

Reply:
0327d28001ec6b4565c662573d841a5fe25ba3bf